### PR TITLE
fix exports

### PR DESCRIPTION
**Pull request by MarcNeely**
These methods need to be exported for use by another project.

### CODE REVIEW

**Comment by pmconne** [comment]
Thanks @MarcNeely.
@danieliborra @markschlosseratbentley if that other project is going to continue to reside in its current repository, we should update the build scripts to build it.

File: iModelCore/iModelPlatform/PublicAPI/DgnPlatform/Render.h [hunk 1]
```diff
@@ -621,8 +621,8 @@ struct TextureMapping
     struct Trans2x3
     {
         double m_val[2][3];
-        Trans2x3() {}
-        Trans2x3(double t00, double t01, double t02, double t10, double t11, double t12)
+        DGNPLATFORM_EXPORT Trans2x3() {}
```
Comment (pmconne) [inline iModelCore/iModelPlatform/PublicAPI/DgnPlatform/Render.h:624]: Did you find these necessary? It won't hurt anything, but functions defined inline shouldn't need an export attribute.

**Review by pmconne — APPROVED** [review]

File: iModelCore/iModelPlatform/PublicAPI/DgnPlatform/Render.h (hunk 1 above)
Comment (MarcNeely) [reply iModelCore/iModelPlatform/PublicAPI/DgnPlatform/Render.h:624]: Yes, you're right.  I misread the error & thought the constructor was what it was complaining about, but it was the FromJason.  I'll remove it from the constructors & test it out to make sure.

Comment (MarcNeely) [reply iModelCore/iModelPlatform/PublicAPI/DgnPlatform/Render.h:624]: OK, Done.

**Comment by danieliborra** [comment]
> Thanks @MarcNeely. @danieliborra @markschlosseratbentley if that other project is going to continue to reside in its current repository, we should update the build scripts to build it.

You mean having a pipeline in imodel-native that compiles imodel-native AND imodel-native-internal, right?

**Pull request merged**